simplekv/transform/decorator.py

```diff
@@ -0,0 +1,80 @@
+# coding=utf8
+
+from .adapter import ReadAdapter, WriteAdapter
+from .transformer import PipeTransformerPair
+
+
+class ValueTransformingDecorator(object):
```

Comment:
What is the reason to not inherit from `.decorator.StoreDecorator`? If a user wants to check for something `KeyValueStore`-like, they would have to explicitly check for this type, too, in addition to `KeyValueStore` and `StoreDecorator`.